### PR TITLE
CEDS-2679 Update content for CRC code 102

### PR DESCRIPTION
**Pull request by janeturner**
CRC code 102 content changed to 'Declaration has not arrived'

### CODE REVIEW

File: app/models/viewmodels/decoder/CRCCode.scala [hunk 1]
```diff
@@ -28,10 +28,10 @@ sealed abstract class CRCCode(override val code: String, override val messageKey
 
 object CRCCode {
 
-  val codes: Set[CRCCode] = Set(Success, PrelodgedDeclarationNotArrived, UCRDoesNotExist)
+  val codes: Set[CRCCode] = Set(Success, PrelodgedDeclarationNotArrived, DeclarationNotArrived)
 
   case object Success extends CRCCode(code = "000", messageKey = "decoder.crc.Success")
   case object PrelodgedDeclarationNotArrived extends CRCCode(code = "101", messageKey = "decoder.crc.PrelodgedDeclarationNotArrived")
-  case object UCRDoesNotExist extends CRCCode(code = "102", messageKey = "decoder.crc.UCRDoesNotExist")
+  case object DeclarationNotArrived extends CRCCode(code = "102", messageKey = "decoder.crc.DeclarationNotArrived")
```
Comment (MaciejRewera) [inline app/models/viewmodels/decoder/CRCCode.scala:35]: 👍

**Review by MaciejRewera — APPROVED** [review]
Big plus for changing the code object's name!

**Comment by MaciejRewera** [comment]
Test this please

**Comment by janeturner** [comment]
Test this please

**Pull request merged**